allensdk/test/brain_observatory/behavior/test_behavior_lims_api.py

```diff
@@ -221,11 +221,12 @@ def test_get_experiment_date(MockBehaviorLimsApi):
 
 
 def test_get_running_speed(MockBehaviorLimsApi):
-    expected = RunningSpeed(timestamps=[0.0, 0.1, 0.2],
-                            values=[8.0, 15.0, 16.0])
+    expected = pd.DataFrame({
+        "timestamps": [0.0, 0.1, 0.2],
+        "values": [8.0, 15.0, 16.0]})
     api = MockBehaviorLimsApi
     actual = api.get_running_speed()
-    assert expected == actual
+    assert expected.to_dict() == actual.to_dict()
```

Comment:
Since these are dataframe now, I think you can use `pd.testing.assert_frame_equal` which is a bit more robust for deeply nested data compared to `to_dict()`.

Reply:
fixed